improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -1194,15 +1196,29 @@ def test_unmatched_periods(self):
         msg = (
             "Period diagnostics with different periods have been provided as "
             "input to the weather symbols code. Period diagnostics must all "
-            "describe the same period to be used together."
-            "['probability_of_number_of_lightning_flashes_per_unit_area_in_"
-            "vicinity_above_threshold: 3600', 'probability_of_shower_condition_"
-            "above_threshold: 7200', 'probability_of_shower_condition_above_"
-            "threshold: 7200']"
+            "describe the same period to be used together.\n"
+            "\['probability_of_shower_condition_above_threshold: 7200', "
+            "'probability_of_number_of_lightning_flashes_per_unit_area_in_"
+            "vicinity_above_threshold: 3600', 'probability_of_lwe_graupel_and_"
+            "hail_fall_rate_in_vicinity_above_threshold: 3600', "
+            "'probability_of_lwe_precipitation_rate_max_above_threshold: 3600', "
+            "'probability_of_shower_condition_above_threshold: 7200'\]"
         )
-        with self.assertRaises(ValueError, msg=msg):
+        with self.assertRaisesRegex(ValueError, msg):
```

Comment:
Previously implemented incorrectly. Fixed in this PR, but unrelated to the change being implemented.